docs/_data/docstoc.yml

```diff
@@ -51,11 +51,11 @@
 
 - title: 'Using Codewind remotely'
   children:
-    - title: 'Overview'
+    - title: 'Overview: Using Codewind remotely'
       url: remote-jane-overview.html
     - title: 'Connecting your IDE to remote Codewind'
       children:
-      - title: 'Connecting your VS Code to remote Codewind'
+      - title: 'Connecting VS Code to remote Codewind'
         url: remotedeploy-vscode.html
       - title: 'Connecting Eclipse to remote Codewind'
```

Comment:
Fiile version is "Connecting Eclipse to your ...." we'll have to make a file change https://github.com/eclipse/codewind-docs/blob/master/docs/_documentations/remotedeploy-eclipse.md